features/user_login.feature

```diff
@@ -0,0 +1,24 @@
+Feature: As a user
+In order to access the app when I already have an account
+I should be able to login
+
+Background:
+  Given the user "Bob" exist
+
+Scenario: User Login
+  When I visit "landing" page
+  And I click "Login" button
+  Then I should be on "Log in" page
```

Comment:
I would avoid keeping this assertion. It is okay while you are working on a scenario but at the end, I would suggest to only keep the last assertions around. Think about it, the next 3 steps ire kind of indirectly asserting that you are on a login page by filling in the fields and pressing the button, right? So this line (11) is not really needed. The endgame here is to see "Signed in successfully.", right? So that is the assertion you should have.